x/minfee/params.go

```diff
@@ -29,11 +29,13 @@ type Params struct {
 	GlobalMinGasPrice sdk.Dec
 }
 
-// RegisterMinFeeParamTable attaches a key table to the provided subspace if it doesn't have one
-func RegisterMinFeeParamTable(ps paramtypes.Subspace) {
-	if !ps.HasKeyTable() {
-		ps.WithKeyTable(ParamKeyTable())
+// RegisterMinFeeParamTable attaches a key table to the provided subspace if it
+// doesn't have one.
+func RegisterMinFeeParamTable(subspace paramtypes.Subspace) paramtypes.Subspace {
+	if !subspace.HasKeyTable() {
+		return subspace.WithKeyTable(ParamKeyTable())
 	}
+	return subspace
```

Comment:
[blocking]
Does the method make changes directly on the `subspace` or it is copied and needs to be returned. If it's the latter, then this also should change:

https://github.com/celestiaorg/celestia-app/blob/ef67d05690d445758472b50ecacaf8a5d5f2b0f2/test/tokenfilter/setup.go#L247

Reply:
can do.

Reply:
825f9263ddd54b62a0fb6c1f7ec1f09d0dfd561e